lib/tapioca/dsl/compiler.rb

```diff
@@ -17,7 +17,7 @@ class Compiler
       include Runtime::Reflection
       extend Runtime::Reflection
 
-      ConstantType = type_member(upper: Module)
+      ConstantType = type_member {{upper: Module}}
```

Comment:
This is the same formatting that sorbet's autocorrect uses but it currently violates our rubocop rules.

Reply:
What's the violation? Spaces between the curly braces?

Reply:
Good question. It mostly works. It's no longer a typecheck error so Sorbet doesn't try to remove the spaces. Funnily enough sometimes the [BindingConstantWithoutTypeAlias cop](https://github.com/Shopify/rubocop-sorbet/blob/2fce9e3c7be251b67acb8a17fd424faff0fb7e54/lib/rubocop/cop/sorbet/binding_constants_without_type_alias.rb#L18) autocorrect is triggered and we insert an incorrect `T.type_alias` with rubocop which would need fixing.

My biggest concern with this is that it's a 2 step process. If we can't change the rubocop rule we should lobby for adding spaces in Sorbet autocorrect.

Reply:
I think that's going to be a very hard sell, given that the Sorbet folks prefer the alternative style of no spaces around braces. If we can auto-correct this once, let's try to do it.

Reply:
I am not that worried about the Sorbet autocorrect, since it is mostly a helper to get the code semantically right, not necessarily stylistically correct.

In previous autocorrects, as well, we've had to apply the autocorrects and then run RuboCop to fix violations. So, let's do that again.

Reply:
I updated this PR to disable the `BindingConstantWithoutTypeAlias` for now.